### ISSUE TITLE
Training loss showed me nan

### ISSUE THREAD

**Issue by nashurgessa** [Stick-To/FCOS-tensorflow#8]
When I train my own data, the loss shows me nan, what is the problem?
